Intrepid.podspec

```diff
@@ -35,7 +35,7 @@ Pod::Spec.new do |s|
 
   s.subspec "Rx" do |rx|
     rx.source_files = "SwiftWisdom/Rx/**/**/*.swift"
-    rx.dependency 'RxSwift', '~> 3.0'
-    rx.dependency 'RxCocoa', '~> 3.0'
+    rx.dependency 'RxSwift', '~> 4.0.0-alpha'
+    rx.dependency 'RxCocoa', '~> 4.0.0-alpha'
```

Comment:
Isn't the latest release beta? Why is this going back to alpha?

Reply:
Being as lenient as possible in the podspec. I know personally of a project that is temporarily stuck using `alpha` because of another dependency so we should allow that too (since we technically can). One can still declare `beta` in their podfile to get the latest.

Reply:
I see